libs/common/src/lib/select-account-kb/select-account-kb.service.ts

```diff
@@ -12,18 +14,54 @@ export class SelectAccountKbService {
 
   readonly accounts = this.accountsSubject.asObservable();
   readonly kbs = this.kbsSubject.asObservable();
+  readonly standalone = this.environment.standalone;
 
-  constructor(private sdk: SDKService) {}
+  constructor(
+    private sdk: SDKService,
+    private toast: SisToastService,
+    @Inject('staticEnvironmentConfiguration') private environment: StaticEnvironmentConfiguration,
+  ) {}
 
   loadAccounts(): Observable<Account[]> {
-    return this.sdk.nuclia.db.getAccounts().pipe(tap((accounts) => this.accountsSubject.next(accounts)));
+    const loadAccountRequest: Observable<Account[]> = this.standalone
+      ? of([standaloneSimpleAccount])
+      : this.sdk.nuclia.db.getAccounts();
+    return loadAccountRequest.pipe(tap((accounts) => this.accountsSubject.next(accounts)));
   }
 
   loadKbs(accountSlug: string): Observable<IKnowledgeBoxItem[]> {
-    return this.sdk.nuclia.db.getKnowledgeBoxes(accountSlug).pipe(tap((kbs) => this.kbsSubject.next(kbs)));
+    const loadKbsRequest: Observable<IKnowledgeBoxItem[]> = this.standalone
+      ? this.sdk.nuclia.db.getStandaloneKbs().pipe(
+          map((kbs) =>
+            kbs.map((kb) => ({
+              id: kb.uuid,
+              slug: kb.uuid,
+              zone: 'local',
+              title: kb.slug,
+              role_on_kb: 'SOWNER' as KBRoles,
+            })),
+          ),
+          catchError((error) => {
+            this.toast.error(
+              'We cannot load your knowledge box, please check NucliaDB docker image is running and try again.',
```

Comment:
Docker is not the only way to run NucliaDB (it can be a Python install for example, and in the future it might be a service in a Linux distibution), so let's be more neutral:
```
We cannot load your knowledge box, please check NucliaDB server is running and try again.```